js/game.js

```diff
@@ -30,7 +31,36 @@ export default class Game {
             sleepingButton: document.querySelector(`${sleepingButton}`),
             playingButton: document.querySelector(`${playingButton}`),
         });
+
+        document.querySelector(`${restartButton}`).addEventListener(
```

Comment:
**nitpick:** This can be dangerous because you don't have handled situation where you have such element found in HTML structure. Typescript will force to change it later anyway :D

Reply:
Ah, nevermind - I thought this code is in the `constructor`